### PR TITLE
Add tctl config feature

### PR DESCRIPTION
**Pull request by feedmeapples**
**What changed?**
Adds config feature to tctl

```bash
./tctl config set version next
```

image

**Why?**
Will allow to switch to experimental `next` version of tctl UX by running

**How did you test it?**
```bash
./tctl config set version next
```
and checking config file

**Potential risks**
no risks

**Is hotfix candidate?**
no

### CODE REVIEW

File: tools/cli/config.go [hunk 1]
```diff
@@ -0,0 +1,114 @@
+// The MIT License
```
Comment (feedmeapples) [inline tools/cli/config.go:1]: this file is a copy of https://github.com/temporalio/tctl/blob/main/cli/config.go

File: go.mod [hunk 2]
```diff
@@ -21,37 +16,24 @@ require (
...
-	github.com/prometheus/procfs v0.6.0 // indirect
-	github.com/rcrowley/go-metrics v0.0.0-20201227073835-cf1acfcdf475 // indirect
-	github.com/rivo/uniseg v0.2.0 // indirect
 	github.com/robfig/cron v1.2.0
-	github.com/russross/blackfriday/v2 v2.1.0 // indirect
-	github.com/sirupsen/logrus v1.8.1 // indirect
 	github.com/stretchr/testify v1.7.0
 	github.com/temporalio/ringpop-go v0.0.0-20200708034907-1e016ebb537a
-	github.com/uber-common/bark v1.3.0 // indirect
+	github.com/temporalio/tctl v0.0.0-20210722235117-a400f112efa4
```
Comment (wxing1292) [inline go.mod:36]: tctl now has its own repo?

Comment (wxing1292) [reply go.mod:36]: or new version has its own repo?

**Review by wxing1292 — APPROVED** [review]
what is the behavior if target config does not exists?

File: go.mod (hunk 2 above)
Comment (feedmeapples) [reply go.mod:36]: new version has its own repo

**Comment by feedmeapples** [comment]
> what is the behavior if target config does not exists?

the code is going to try to create it under `<user_home_dir>/.config/temporalio/tctl.yml`

**Pull request merged**